### PR TITLE
[GNA] add split_to_concat_with_3_inputs test

### PR DESCRIPTION
**Pull request by a-noskov**
### Details:
 - rewrote split_to_concat_with_3_inputs test from dldt_tests_closed repo

### Tickets:
 - *52043*

### CODE REVIEW

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/split_to_concat_with_3_inputs.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2021 Intel Corporation
```
Comment (elilobanova) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/split_to_concat_with_3_inputs.cpp:1]: 2018 is extra for a new file
the same is fore other added files

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/split_to_concat_with_3_inputs.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+namespace SubgraphTestsDefinitions {
+namespace {
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16
+};
+
+std::map<std::string, std::string> additional_config = {};
+//const std::vector<std::map<std::string, std::string>> additional_config = {
```
Comment (elilobanova) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/split_to_concat_with_3_inputs.cpp:18]: why is it commented?

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/split_to_concat_with_3_inputs.cpp [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    auto params = ngraph::builder::makeParams(ngPrc, { inputShape });
+
+    auto split = ngraph::builder::makeSplit(params[0], ngPrc, 3, 1);
+    auto relu1 = std::make_shared<ngraph::opset3::Relu>(split->output(0));
+    auto tanh1 = std::make_shared<ngraph::opset3::Tanh>(split->output(1));
+
+    auto concat = std::make_shared<ngraph::opset1::Concat>(ngraph::OutputVector{relu1, tanh1, split->output(2)}, 1);
+    auto relu2 = std::make_shared<ngraph::opset3::Relu>(concat);
+
+//    ngraph::ResultVector results{ std::make_shared<ngraph::op::Result>(relu1), std::make_shared<ngraph::op::Result>(relu2) };
```
Comment (elilobanova) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/split_to_concat_with_3_inputs.cpp:43]: please, delete the commented code

**Review by elilobanova — APPROVED** [review]

**Comment by a-noskov** [comment]
PR closed due to existing new one https://github.com/openvinotoolkit/openvino/pull/7775

**Pull request closed without merging**